service/dap/server.go

```diff
@@ -2057,8 +2060,11 @@ func (s *Server) onEvaluateRequest(request *dap.EvaluateRequest) {
 			}
 		}
 		var opts convertVariableFlags
-		if ctxt == "variables" || ctxt == "hover" || ctxt == "clipboard" {
-			opts |= showFullValue
+		if ctxt == "clipboard" || ctxt == "variables" {
```

Comment:
With clipboard option on, when do we get "variables"? Is this in case other editors don't use clipboard?

Reply:
This is to support clients that do not use clipboard context